src/test/isolation2/input/resgroup/resgroup_startup_memory.source

```diff
@@ -0,0 +1,75 @@
+-- start_ignore
+! gpconfig -c runaway_detector_activation_percent -v 100;
+! gpstop -rai;
+
+CREATE OR REPLACE FUNCTION resGroupPalloc(float) RETURNS int AS
+'@abs_builddir@/../regress/regress@DLSUFFIX@', 'resGroupPalloc'
+LANGUAGE C READS SQL DATA;
+-- end_ignore
+
+ALTER RESOURCE GROUP default_group SET memory_limit 10;
+ALTER RESOURCE GROUP admin_group SET memory_limit 30;
+
+CREATE RESOURCE GROUP test_group with (concurrency=1, memory_limit=59, memory_spill_ratio=100, cpu_rate_limit=20);
+CREATE ROLE test_role RESOURCE GROUP test_group;
+CREATE ROLE default_role RESOURCE GROUP default_group;
+
+-- 1. Occupy all memory on a segment and fail with vmem tracker error
+ALTER RESOURCE GROUP admin_group SET concurrency 1;
+ALTER RESOURCE GROUP default_group SET concurrency 1;
+
+-- Start a session which will be detached from a group when the query is done
+-- resource groups can't see startup chunks occupied by a detached session but
+-- the vmem tracker can
+0: SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE pid != (SELECT pg_backend_pid());
```

Comment:
here and below
```suggestion
0: SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE pid != pg_backend_pid();
```
Do we really want to kill ALL backends except the current one?!

Reply:
Yes, I really want ALL backend except the current one, that's the point of the test. I can change it if you explain what problems it can cause